pytorch_toolkit/action_recognition_2/model_templates/gesture_recognition/s3d-rgb-mobilenet-v3-stream-jester/template.yaml

```diff
@@ -0,0 +1,62 @@
+name: s3d-rgb-mobilenet-v3-stream-jester
+domain: Action Recognition 2
+problem: Gesture Recognition
+framework: OTEAction
```

Comment:
Please create a tag in corresponding repo using semantic versioning and put it here. You can have a look at openvinotoolkit/mmdetection tags.

Reply:
Done.